### PR TITLE
chore(menu): convert to lightningcss

### PR DESCRIPTION
**Pull request by najikahalsema**
A draft to test if it passes VRTs
## Description

Convert menu to lightningcss config in order to prepare for core tokens work. 

## Related issue(s)

-

## How has this been tested?

VRTs 

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: packages/menu/src/spectrum-config.js [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+                {
+                    collapseSelector: true,
+                    find: [
+                        // this doesn't work quite right
+                        builder.class('spectrum-Menu'),
+                        builder.combinator('descendant'),
+                        builder.class('spectrum-Menu-divider'),
+                        // builder.combinator(' '),
+                    ],
+                    replace: [
```
[the remark this answers is not in the document]
Comment (najikahalsema) [reply packages/menu/src/spectrum-config.js:167]: It works now! That's before we figured out how to spell 😉

**Comment by najikahalsema** [comment]
> CI says you didn't run `yarn`/`yarn process-spectrum` before you last commit.

I'm working on it!!! There's still some things left to convert with that whole `host:not(disabled)` whatever

**Review by Westbrook — CHANGES_REQUESTED** [review]
LGTM. Please pull out the `spectrum-config.v1.js` file and then we'll merge this in!

**Review by Westbrook — APPROVED** [review]
Love to see it!

Here we mark the passing of a good friend who got us through many a difficult Spectrum CSS upgrades, but left of confused and with more work than we wanted it their old age. Long live the king!

**Pull request merged**